src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp

```diff
@@ -53,16 +55,22 @@ namespace argos
       m_cVelocityPrev.Set(0.0, 0.0, 0.0);
       /* reset the drone's angular velocity */
       m_cAngularVelocity.Set(0.0, 0.0, 0.0);
-      m_cAngularVelocityPrev.Set(0.0, 0.0, 0.0);
+      m_cPrevAngularVelocity.Set(0.0, 0.0, 0.0);
```

Comment:
Any reason for putting `Prev` at the beginning? I think it is more readable code-wise to have it at the end...